pkg/datapath/linux/ipsec/ipsec_linux.go

```diff
@@ -520,11 +520,6 @@ func ipSecReplaceStateIn(log *slog.Logger, params *IPSecParameters) (uint8, erro
 			Value: 0,
 			Mask:  linux_defaults.OutputMarkMask,
 		}
-	} else if params.ReqID == EncryptedOverlayReqID {
-		state.OutputMark = &netlink.XfrmMark{
-			Value: linux_defaults.RouteMarkDecryptedOverlay,
```

Comment:
Should we also delete the definition of `RouteMarkDecryptedOverlay`? I think that was the only usage.

Reply:
Yup, will do.

Reply:
Done.